.github/workflows/scan-vulns.yaml

```diff
@@ -0,0 +1,68 @@
+name: scan_vulns
+on:
+  push:
+    paths-ignore:
+      - "docs/**"
+      - "library/**"
+      - "**.md"
+  pull_request:
+    paths-ignore:
+      - "docs/**"
+      - "library/**"
+      - "**.md"
+  workflow_dispatch:
+
+permissions: read-all
+
+jobs:
+  govulncheck:
+    name: "Run govulncheck"
+    runs-on: ubuntu-22.04
+    timeout-minutes: 15
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@63c24ba6bd7ba022e95695ff85de572c04a18142 # v2.7.0
```

Comment:
side note: we should looking into adding harden runner for all our workflows

Reply:
issue created : https://github.com/deislabs/ratify/issues/1525